recipes/triqs_cthyb/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "2.2.1" %}
+
+package:
+  name: triqs_cthyb
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/cthyb/releases/download/{{ version }}/cthyb-{{ version }}.tar.gz
+  sha256: de78b39a354c0e1344987c26403919c23611635ab3d75620aa01bd5b06431bdc
+
+build:
+  number: 0
+  skip: True  # [win or py>30]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - triqs {{ '.'.join(version.split('.')[:2]) }}
+    - boost-cpp
+    - nfft
+    - hdf5
+    - openmpi
```

Comment:
MPI should be handled as here: https://conda-forge.org/docs/maintainer/knowledge_base.html#message-passing-interface-mpi

Reply:
Dear @scopatz,
I have changed the recipe to depend on `{{ mpi }}` in `host` and `run` and added a `conda_build_config.yaml` file with the content
```yaml
mpi:
  -mpich
  -openmpi
```

I have done the same for the [triqs-feedstock](https://github.com/conda-forge/triqs-feedstock)

This seems to have introduced various package conflicts that I do not quite understand however.

Reply:
I didn't wait long enough after updating triqs for the index to update. Everything seems to be fine in fact.